autoprotocol/instruction.py

```diff
@@ -250,7 +257,22 @@ def __init__(self, groups, magnetic_head):
                 f"container_types: {self.heads[magnetic_head]} for head_type: "
                 f"{magnetic_head}"
             )
-
+        non_valid_well_volumes = [
+            w
+            for _ in containers
```

Comment:
`_` is typically used when the return will not be used. In this case it is valuable to know what `_` represents